packages/minicaml/minicaml.0.3.2/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+maintainer: "sudo-woodo3@protonmail.com"
+authors: ["Alessandro Cheli"]
+homepage: "https://github.com/0x0f0f0f/minicaml"
+bug-reports: "https://github.com/0x0f0f0f/minicaml/issues"
+dev-repo: "git+https://github.com/0x0f0f0f/minicaml.git"
+license: "MIT"
+synopsis: "A simple, didactical, purely functional programming language"
+description: "A simple, didactical, purely functional programming language written for the programming 2 course at the University of Pisa, extended with a simple parser made with Menhir and ocamllex"
+doc: "https://github.com/0x0f0f0f/minicaml"
+build: [
+  [make "-j" jobs]
```

Comment:
```suggestion
  ["dune" "build" "-p" name "-j" jobs]
```

Reply:
This command should be used in most cases for releases. The commands you are calling in your Makefile are failing on MacOS and also might break in the future (`dune` without `-p` or `--profile=release` enables some warnings as errors by default)

Reply:
I'm using sed in the Makefile to disable the bisect_ppx preprocessor when not running `make test`. I've seen there was a mess in Github issue discussion related to bisect_ppx usage with dune. Would you suggest any way of doing so by using only dune and without having to edit the file before build? So I may get rid of calling the makefile from the opam file. Thanks a lot.

Reply:
I was saying that because I tried running your make command and it did not change the dune file in any way. The sed command does nothing to the file and `bisect_ppx` is not used since it is already commented out behind the `;` (https://dune.readthedocs.io/en/stable/lexical-conventions.html#comments)

Reply:
>  Would you suggest any way of doing so by using only dune and without having to edit the file before build?

Yes, also I think that would be cleaner and better for you and everyone else.

Reply:
`bisect_ppx` is commented out in the dune file when running the default make target. it gets uncommented by sed when running `make test`. dune's documentation is quite messy and there is nothing written in there about enabling certain s-expressions only when tests are enabled (or how to emulate that using profiles). Please let me know if you know any workaround.  

https://github.com/ocaml/dune/issues/57

Reply:
From the documentation, using `bisect_ppx --conditional` should be enough: https://github.com/aantron/bisect-starter-dune/blob/master/dune#L4